src/accountsdb/db.zig

```diff
@@ -438,24 +438,14 @@ pub const AccountsDB = struct {
 
         self.logger.infof("[{d} threads]: reading and indexing accounts...", .{n_parse_threads});
         {
-            var handles = std.ArrayList(std.Thread).init(self.allocator);
-            defer {
-                for (handles.items) |*h| h.join();
-                handles.deinit();
-            }
-
-            try spawnThreadTasks(
-                &handles,
-                loadAndVerifyAccountsFilesMultiThread,
-                .{
-                    loading_threads.items,
-                    accounts_dir,
-                    snapshot_manifest.file_map,
-                    accounts_per_file_estimate,
-                },
-                n_account_files,
-                n_parse_threads,
-            );
+            var wg: std.Thread.WaitGroup = .{};
+            defer wg.wait();
+            try spawnThreadTasks(&wg, n_account_files, n_parse_threads, null, loadAndVerifyAccountsFilesMultiThread, .{
+                loading_threads.items,
+                accounts_dir,
+                snapshot_manifest.file_map,
+                accounts_per_file_estimate,
+            });
```

Comment:
can we add another comma to the end of all these calls to make the parameters organized better - its a bit long rn

```suggestion
            try spawnThreadTasks(&wg, n_account_files, n_parse_threads, null, loadAndVerifyAccountsFilesMultiThread, .{
                loading_threads.items,
                accounts_dir,
                snapshot_manifest.file_map,
                accounts_per_file_estimate,
            },);
```